portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,9 +14,14 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.MyComments;
```

Comment:
nit - alphabetize these imports - or it actually looks like MyComments is unused in this PR. Revert if that's the case.

Reply:
yes it is unused now :) thanks! updated!